### PR TITLE
Typo "Azure blob storage"→"Azure Blob Storage"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://docs.microsoft.com/en-us/azure/architecture/best-practices/data-partitioning-strategies
#PingMSFTDocs

### CODE REVIEW

File: docs/best-practices/data-partitioning-strategies.md [hunk 1]
```diff
@@ -117,9 +117,9 @@ Consider the following points when you design your entities for Azure table stor
 
 For more information, see [Azure storage table design guide] and [Scalable partitioning strategy].
 
-## Partitioning Azure blob storage
+## Partitioning Azure Blob Storage
 
-Azure blob storage makes it possible to hold large binary objects. Use block blobs in scenarios when you need to upload or download large volumes of data quickly. Use page blobs for applications that require random rather than serial access to parts of the data.
+Azure Blob Storage makes it possible to hold large binary objects. Use block blobs in scenarios when you need to upload or download large volumes of data quickly. Use page blobs for applications that require random rather than serial access to parts of the data.
```
Comment (ckittel) [inline docs/best-practices/data-partitioning-strategies.md:122]: It's actually "Azure Blob storage" (lower s in storage). Any chance you can update this PR?  Thanks!

**Review by ckittel — CHANGES_REQUESTED** [review]

**Comment by PRMerger18** [comment]
@EdPrice-MSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by EdPrice-MSFT — DISMISSED** [review]
Good update. I edited the casing.

**Comment by EdPrice-MSFT** [comment]
#Sign-Off

**Review by ckittel — APPROVED** [review]

**Comment by ktoliver** [comment]
Hi folks - The current guidance in the Microsoft-internal cloud style guide is "Blob Storage" 

First mention: Azure Blob Storage

Subsequent mentions: Blob Storage

(https://styleguides.azurewebsites.net/Styleguide/Read?id=2696&topicid=39162 - internal link)

**Comment by PRMerger17** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by ktoliver — APPROVED** [review]

**Pull request merged**